test/config.js

```diff
@@ -20,6 +20,9 @@
     if (!('Promise' in window)) {
         deps.push('polyfills/promise');
     }
+    if (!('VTTCue' in window)) {
+        deps.push('polyfills/vttcue');
+    }
```

Comment:
Tests still failing in commercial. In the commercial test/qunit/config.js add the vttcue polyfill here:

```js
var deps = [
        // Add polyfills for IE9
        'os/polyfills/base64',
        'os/polyfills/promise'
    ];
```